src/pages/cli/migration/previous-versions.mdx

```diff
@@ -0,0 +1,26 @@
+export const meta = {
+  title: `Using previous CLI versions`,
+  description: `Resolve issues related to previous CLI versions`,
+};
+
+<Callout warning>
+
+#### If you are an existing Amplify CLI customer that installs the CLI using NPM, continue reading.
+
+On January 8th, 2022, a malicious commit was made to [colors.js](https://www.npmjs.com/package/colors/v/1.4.2) which is a dependency of Amplify CLI. This commit causes an infinite string of random characters to be printed to the terminal when running any CLI command. The issue has been patched in version 7.6.7 of the CLI. However, if you need to install a previous version of the CLI, it will not contain the patch. To install a previous version of the CLI, you can do one of the following:
```

Comment:
It looks like npm has unpublished the malicious versions. Do we still need to call out `colors` specifically?

Reply:
gonna close this now that the issue is fixed on npm